src/web/components/Card/Card.tsx

```diff
@@ -178,9 +179,25 @@ export const Card = ({
                                                     }
                                                 />
                                             </>
-                                        )}
-                                    </>
-                                </CardFooter>
+                                        ) : (
+                                            undefined
+                                        )
+                                    }
+                                    commentCount={
+                                        longCount && shortCount ? (
```

Comment:
the way `formatCount` was written means this will always be true, because if no count is added `longCount = ''` and  `shortCount = ''`

Reply:
'' resolves to false my bad

Reply:
Actually, `''` is falsy 🤷‍♂